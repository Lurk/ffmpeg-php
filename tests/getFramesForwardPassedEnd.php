<?php
$mov = new ffmpeg_movie(dirname(__FILE__) . '/test_media/robot.avi');

$framecount = $mov->getFrameCount();
for($i = 1; $i <= $framecount + 1; $i++) {
    $img = sprintf("%s/test-%04d.png", dirname(__FILE__), $i);
    $frame = $mov->getFrame($i);
    if ($frame == false) {
        break;
    }
    $image = $frame->toGDImage();
    imagepng($image, $img);
    printf("ffmpeg getFrame($i): md5 = %s\n", md5(file_get_contents($img)));
    imagedestroy($image);
    unlink($img);
}
?>
