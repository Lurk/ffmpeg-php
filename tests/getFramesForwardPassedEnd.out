Fatal error: failed to convert frame to gd image in /src/ffmpeg-php-code/ffmpeg-php/tests/getFramesForwardPassedEnd.php on line 11